docs/en/modules/ws2812.md

```diff
@@ -23,7 +23,11 @@ Initialize UART0 (TXD0) too if `ws2812.MODE_DUAL` is set.
 `ws2812.init([mode])`
 
 #### Parameters
-- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted) or `ws2812.MODE_DUAL`
+- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted), `ws2812.MODE_SINGLE_OPEN`, or `ws2812.MODE_DUAL`
+
+In `ws2812.MODE_SINGLE_OPEN` you will have to pull the WS2812 data line
+high.  As the ESP8266 is 5V tolerant but 3.3V may not rise to the logic high
+level of a WS2812, this arrangement may increase reliability of the signal.
```

Comment:
This paragraph could do with fleshing out to the high quality of your PR description.